### PR TITLE
Hw liquibase

### PR DESCRIPTION
**Pull request by ChabVlad**

### CODE REVIEW

File: src/main/java/project/bookstore/repository/BookRepository.java [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
 
-import java.util.List;
-import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
 import project.bookstore.model.Book;
 
-public interface BookRepository {
-    Book save(Book book);
+@Repository
```
Comment (MentorMateAcademy) [inline src/main/java/project/bookstore/repository/BookRepository.java]:
```suggestion
```

**Review by MentorMateAcademy — CHANGES_REQUESTED** [review]

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
you should implement 
PUT: /api/books/{id} (Update a specific book)
DELETE /api/books/{id} (Delete a specific book)

See task description

File: src/main/java/project/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -31,7 +33,20 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestBookDto) {
-        return bookMapper.toDto(bookService.save(requestBookDto));
+    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+        return bookMapper.toDto(bookService.save(requestDto));
```
Comment (Elena-Bruyako) [inline src/main/java/project/bookstore/controller/BookController.java:37]:
```suggestion
        return bookService.save(requestDto);
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
one comment

**Review by sarakhmen — APPROVED** [review]

**Pull request merged**